### ISSUE TITLE
manual

### ISSUE THREAD

**Issue by galapah** [jandrewrfarrell/RUFUS#2]
Hello,

I am keen on trying this tool on our data! I already have kmers calculated by Jellyfish. However, I am not sure how to proceed further. Can you please provide at least a sample sequence of commands?

Jan

**Comment by jandrewrfarrell** [comment]
This project is still under development, but you will find script files
that should run everything in RUFUS/scripts/.  RunRUFUS.HumanTrio.sh should
run a trio analysis start to finish.  What type of experiment are you
trying to run ?

**Comment by galapah** [comment]
Hi,

thank you for quick response!
I am sorry, I cannot see the RunRUFUS.HumanTrio.sh
http://runrufus.humantrio.sh/ even on GitHub:
https://github.com/jandrewrfarrell/RUFUS/tree/master/scripts

I have reads from two plant individuals (mutant vs wildtype).

Jan

2016-06-16 22:11 GMT+02:00 J Andrew R Farrell notifications@github.com:

> This project is still under development, but you will find script files
> that should run everything in RUFUS/scripts/. RunRUFUS.HumanTrio.sh should
> run a trio analysis start to finish. What type of experiment are you
> trying to run ?
>

**Comment by galapah** [comment]
I am running scripts/RunTumor.sh now.

2016-06-16 22:23 GMT+02:00 Jan Hapala jan@hapala.cz:

**Comment by jandrewrfarrell** [comment]
yeah, tumor normal is what you should use for mutant vs wild type

**Comment by jandrewrfarrell** [comment]
do you have a reference for the plant your using? If so you will need to
change that in OverlapBashMultithread.sh to get a VCF. if you don't you
just want the mutant contigs you are all set

**Comment by galapah** [comment]
Great! Thank your for the tips! It's Arabidopsis so I have a reference.

2016-06-16 22:46 GMT+02:00 J Andrew R Farrell notifications@github.com:
